Packs/Base/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "Base",
     "description": "The base pack for Cortex XSOAR.",
     "support": "xsoar",
-    "currentVersion": "1.31.10",
+    "currentVersion": "1.31.11",
```

CommonServerPython has already been updated,
Please change to:

```suggestion
    "currentVersion": "1.31.12",
```